whirlpool/appliance.py

```diff
@@ -71,6 +75,13 @@ def _create_headers(self):
             "Cache-Control": "no-cache",
         }
 
+    def _check_previous_attribute_value(self, attr, json_msg):
+        """Confirm current stored value is equal to expected previous value."""
+        return (
+            str(json_msg["processedValueMap"]["previousValueMap"][attr])
+            == self._data_dict["attributes"][attr]["value"]
+        )
+
```

Comment:
Besides the eventsocket stopping, it also is exhibiting failure to send "some" messages then starting back up on it's own.  Found that the messages contains a previous value we can use to see if we are up to date - if not, just refresh everything with a fetch.